docs/announcements/github-release-v0.1.73.md

```
@@ -0,0 +1,21 @@
+# 🚀 Release Highlights — v0.1.73 (2026-04-06)
+
+### 🧬 [Eval Criteria Evolver Subagent](https://docs.massgen.ai/en/latest/user_guide/concepts.html)
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)